improver/wxcode/weather_symbols.py

```diff
@@ -100,6 +101,9 @@ def __init__(self, wxtree="high_resolution"):
                 Used to choose weather symbol decision tree.
                 Default is "high_resolution"
                 "global" will load the global weather symbol decision tree.
+            model_id_attr (str):
+                Name of attribute recording source models. The source models
```

Comment:
Needs updating with whatever you choose to put in the CLI equivalent bit

Reply:
Amended as suggested.